### PR TITLE
Release 1.33.0

### PR DESCRIPTION
**Pull request by scrosby**
## Changes proposed in this PR

- Release 1.33.0
- 
-

### CODE REVIEW

File: scheduler/project.clj [hunk 1]
```diff
@@ -13,7 +13,7 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 ;;
-(defproject cook "1.32.2-SNAPSHOT"
+(defproject cook "1.33.0-SNAPSHOT"
```
Comment (pschorf) [inline scheduler/project.clj]: Should be 1.33.1-SNAPSHOT

File: scheduler/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
...
+### Fixed
+- Improved flaky tests, from @pschorf
+### Added
+- Support for multiple submit plugins, from @pschorf
+- Maximum command line length parameter, from @pschorf
+- Improved error logging, from @pschorf
+### Changed
+- Refactors for Kubernetes support, from @scrosby & @pshorf
+- Check quota when rebalancing, from @pschorf
+- Optimize the job fetching code to not round-trip to UUID, from @pschorf
```
Comment (pschorf) [inline scheduler/CHANGELOG.md]: this was you 😄

File: scheduler/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.33.0] - 2019-08-07
+### Fixed
+- Improved flaky tests, from @pschorf
```
Comment (pschorf) [inline scheduler/CHANGELOG.md]: You can drop this line, I usually don't comment on test changes.

File: scheduler/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
...
+### Added
+- Support for multiple submit plugins, from @pschorf
+- Maximum command line length parameter, from @pschorf
+- Improved error logging, from @pschorf
+### Changed
+- Refactors for Kubernetes support, from @scrosby & @pshorf
```
Comment (pschorf) [inline scheduler/CHANGELOG.md]: You can drop this one as well, I usually just focus on user-facing changes.

File: scheduler/CHANGELOG.md (hunk 2 above)
Comment (scrosby) [reply scheduler/CHANGELOG.md]: Yes. It was. :)

**Pull request merged**